README.md

```diff
@@ -25,8 +25,27 @@ La idea del proyecto está basada en resolver los principales problemas a los qu
 - Además, sería muy útil recibir la estadística del historial de ventas de cada producto para estimar el tiempo para la próxima compra y la cantidad necesaria
 
   
-  
-  
+# Guía de instalación y uso
+En primer lugar habrá que descargar el proyecto desde este repositorio, usando la orden:
+  - git clone git@github.com:eantoniocalo18/IV
+
+Para la instalación del proyecto, será necesario tener descargado el software Poetry [aquí] (https://python-poetry.org/docs/)
+En el caso de sistemas Unix se podrá utilizar el siguiente comando 
```

Comment:
Si vais a copiar las instrucciones que hay en poetry, no lo hagáis. No aporta nada ni a vuestra evaluación ni al que lea el README. Tampoco evidencia que entiendas lo que hace esa orden (y vas a necesitar entenderlo para el objetivo 5)